core/src/test/java/io/questdb/test/cairo/CairoEngineTest.java

```diff
@@ -175,11 +175,13 @@ public void testDuplicateTableCreation() throws Exception {
             TableModel model = new TableModel(configuration, "x", PartitionBy.NONE).col("a", ColumnType.INT);
             AbstractCairoTest.create(model);
             try (
-                    Path path = new Path();
-                    MemoryMARW mem = Vm.getMARWInstance()
+                    Path path = new Path()
             ) {
-                engine.createTable(securityContext, mem, path, false, model, false);
-                fail("duplicated tables should not be permitted!");
+                try (MemoryMARW mem = Vm.getCMARWInstance()
+                ) {
```

Comment:
nit: new formatting looks weird here

Reply:
this is how intellij inlines functions :( There were a lot of places like this that i had fixed manually. Missed this one.

Reply:
fixed in https://github.com/questdb/questdb/pull/5300/commits/3c891b637c1a21d111fd2cf31361365a90e44a42